sklearn/datasets/tests/test_20news.py

```diff
@@ -77,6 +79,10 @@ def test_20news_vectorized():
     assert_equal(bunch.target.shape[0], 7532)
     assert_equal(bunch.data.dtype, np.float64)
 
+    # test return_X_y option
```

Comment:
How about the training set?

Reply:
I thought only one subset return_X_y check was needed since fetch_20newsgroups_vectorized is subsetting the fetched dataset before return_X_y is checked.